packages/desktop/src/main/index.ts

```diff
@@ -26,6 +26,10 @@ app.setAppUserModelId("org.aics.alleninstitute.fileexplorer");
 // Prevent window from being garbage collected
 let mainWindow: BrowserWindow | undefined;
 
+// Maximum number of renderer reload attempts
+const MAX_RELOAD_ATTEMPTS = 3;
```

Comment:
Why specifically 3? If we know that for standard (non broken) refreshes it only takes 1 re-reload to work, for any other condition up to 3 probably means something is really broken and we just want it to error?

Reply:
no real reason, 3 is just a standard I've done before for retries

Reply:
I agree lets just fail out after one. Are you seeing it ever continue to fail after one?

Reply:
btw if/once you convert it to 1 retry i'd make this a boolean flag rather than a count for readability.

Reply:
resolved in [45ea20a](https://github.com/AllenInstitute/biofile-finder/pull/447/commits/45ea20a5e2e2f88bd759b9fa4f2922928e7ee2a8)